staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1514,3 +1517,14 @@ message ReopenContainerLogRequest {
 
 message ReopenContainerLogResponse{
 }
+
+message CheckpointContainerRequest {
+    // ID of the container to be checkpointed.
+    string container_id = 1;
+    // Location of the checkpoint archive used for export
+    string location = 2;
+    // Timeout after which the kubelet will assume this request has failed
```

Comment:
```suggestion
    // Timeout in seconds for the checkpoint to complete. Default: 0 (no timeout - run to completion)
```
IMO the default should be either 0 - run to completion or 0 should not be supported at all, make the caller use 1sec or greater timeout. Not sure if using default cri timeout fits the checkpoint use case.  @mrunalp thoughts?

Reply:
Sounds good to me. I made the change.